### DISCUSSION TITLE
Créer un minievent qui fait la promo du canard écarlate

### DISCUSSION THREAD

**Discussion by symsym-1629** [Crownicles/Crownicles#1007] [Nouvelles idées]
.

**Comment by beaud37210** [comment]
Moi j'aime bien du moment que ce n'est pas que pour le canard mais pour un animal aléatoire...
Après je ne suis pas dev donc...

**Comment by symsym-1629** [comment]
Je me suis mal fait comprendre, je parle du futur jeu de draft

**Comment by beaud37210** [comment]
J'ai toujours pas compris

**Comment by BastLast** [comment]
Le jeu ne sortira pas avant plusieurs mois donc aucune urgence pour ce genre de chose
